src/views/Incorporations/ReviewConfirm.vue

```diff
@@ -55,7 +55,7 @@
       </header>
       <div :class="{ 'invalid-section': isCertifyInvalid }">
         <Certify
-          :currentDate="getCurrentDate"
+          :currentDate="dateToYyyyMmDd(getCurrentJsDate)"
```

Comment:
FWIW, I think the old code would have worked here (except maybe on day rollover).

No need to change. I just wanted to point our that both codes display yyyy-mm-dd.